anyPass.ts

```diff
@@ -0,0 +1,23 @@
+import curryN from "./utils/curry_n.ts"
+import { Curry1, Func } from "./utils/types.ts"
+import { max } from './max.ts'
+import { pluck } from './pluck.ts'
+import { reduce } from './reduce.ts'
+
+function _anyPass(preds: any) {
+  return curryN(reduce(max, 0, pluck('length', preds)), function(this: any) {
```

Comment:
same here, readability!

Reply:
Yes, but since length is being used only once, declaring separately will just increase the number of lines.